### ISSUE TITLE
[Enhancement] Add an option to open Steam Links in the Steam Client.

### ISSUE THREAD

**Issue by Jas-SinghFSU** [HerrKnarz/Playnite-Extensions#71]
**Is your feature request related to a problem? Please describe.**
The links that are currently fetched by Metadata sources always open in a browser since they are your standard URLs. Often times, you're logged into Steam and/or maybe you want to install a Workshop item for a Steam game and to do that, you have to log into steam on the browser. Whereas, if I was able to open steam links in steam, I could easily manage all of that.

**Describe the solution you'd like**

Allow an option to open all Steam links with Steam (a checkbox in the settings).

**Describe alternatives you've considered**
A more powerful tool would be an entirely new 'Reformatting' tab for link where you can modify/morph links.

Ex: `https://steamcommunity.com/app/730/discussions/` could be morphed via a regex
```
(.*)steamcommunity.com(.*)
```
into
```
steam://openurl/https://steamcommunity.com/$2
```
Where $2 ($n where n is the matched regex - $1 would be  `https://`) is the 2nd matched regex `/app/730/discussions/`.

But I think this is way out of scope for this specific ask and is probably an entirely new feature request instead.

**Comment by Jas-SinghFSU** [comment]
There is a Microsoft Edge extension that does exactly what I recommended in my alternatives called [Redirector](https://microsoftedge.microsoft.com/addons/detail/redirector/jdhdjbcalnfbmfdpfggcogaegfcjdcfp#:~:text=Redirector%20is%20an%20extension%20that,.com%2Fbar.html.).

![Screenshot 2023-10-10 180719](https://github.com/HerrKnarz/Playnite-Extensions/assets/21210205/7ca3b389-35cb-4976-a951-04bfe3a8da3c)

**Comment by HerrKnarz** [comment]
Hmm, I don't know if it's possible for an extension to change the link once it is clicked. I can add a function to convert already existing links though, and add options to fetch new links directly in the steam app format and also add links to discussions, workshop etc.

**Comment by Jas-SinghFSU** [comment]
Yea, I think that's ideal. Changing links that currently exist and changing new links automatically would solve this. 

Not sure how useful changing links after they're clicked would be, even if possible.

**Comment by HerrKnarz** [comment]
I added the functionality now. It will be available with the 3.6 release shortly.

**Comment by Jas-SinghFSU** [comment]
Man, that is awesome. Thank you!
